### ISSUE TITLE
App crashes on IOS for RN = 0.75.4 

### ISSUE THREAD

**Issue by kcadi** [rive-app/rive-react-native#268]
App crashes on IOS , works fine on Android for RN = 0.75.4 
<img width="1440" alt="Screenshot 2024-10-16 at 6 41 09 PM" src="https://github.com/user-attachments/assets/04157fd2-4f65-4099-ad05-729d257bd39b">

There may be some error in the package itself , as I even commented out all the import statements to check if this issue is replicated , but the issue still lies

**Comment by alextoudic** [comment]
@dskuza I also reported the same issue on https://github.com/rive-app/rive-ios/issues/345#issue-2591999986 with a bit of added context

It looks like a symbol collision between RiveRuntime's Yoga and React Native's Yoga occuring when using static linkage. I thought namespacing Yoga in RiveRuntime, might help, what do you think?

**Comment by PierreCapo** [comment]
As a workaround, downgrading to `rive-react-native: 7.0.5` work on my side 👍

**Comment by kcadi** [comment]
> As a workaround, downgrading to `rive-react-native: 7.0.5` work on my side 👍

Its works on the emulator , but crashes in production

**Comment by hamdij0maa** [comment]
+ 1, also happening in version 8.0.0

**Comment by guval-gh** [comment]
Same with:
```json
{
   "expo": "~52.0.0-preview.19",
   "react-native": "0.76.1",
   "rive-react-native": "^8.0.0"
}
```

**Comment by lancesnider** [comment]
This should be resolved with 8.2.0. 

## Troubleshooting

- Remove `node_modules` and `ios/Pods` 
- Remove any reference to Rive in `package-lock.json` or `yarn.lock` and run `yarn` or `npm i` again
- Remove any reference to Rive in the `ios/Podfile.lock` and rerun `Pod install`

**Comment by lancesnider** [comment]
Sorry, I was wrong about it being fixed in 8.2.0. I just pushed the button to release 8.3. @leemhenson It sounds like it's working for you now? Please let me know if you run into any issues with this.
